src/assets/translations/knb.json

```diff
@@ -33584,7 +33584,7 @@
                   "verse": "6"
                 },
                 {
-                  "text": "Menjetek el, s mondjátok meg tanítványainak és Péternek: »Előttetek megy Galileába, ott majd meglátjátok őt, amint megmondta nektek.«",
+                  "text": "Menjetek el, s mondjátok meg tanítványainak és Péternek: ‘Előttetek megy Galileába, ott majd meglátjátok őt, amint megmondta nektek.’«",
```

Comment:
Taking the API response:
https://szentiras.hu/api/ref/mk16,6-7/KNB
to the correct form, which seen several places in the KNB Bible e.g: KNB [lk19,30 - lk19,31].

Reply:
Documented